app/views/monsters/show.html.erb

```diff
@@ -0,0 +1,3 @@
+<div id="app">
+
+</div>
```

Comment:
Instead of having to create this app div's for every React page, take a look at the comment I left on your routes.rb!

Reply:
done